src/frontend/admin/src/components/presentational/table/TableComponent.tsx

```diff
@@ -50,8 +50,16 @@ export function TableComponent<T>({ enableEdit = true, ...props }: Props<T>) {
           return (
             <TableDefaultActions
               entityName={entityName}
-              onDelete={() => props.onRemoveClick?.(params.row)}
-              onEdit={() => props.onEditClick?.(params.row)}
+              onDelete={
+                props.onRemoveClick
+                  ? () => props.onRemoveClick?.(params.row)
+                  : undefined
```

Comment:
Should work nope ? (Not quit sure maybe it generates a TS error 🤔)
```suggestion
                props.onRemoveClick && () => props.onRemoveClick?.(params.row)
```